props/saber_BC_buttons.h

```diff
@@ -1567,6 +1589,25 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
     if (scroll_presets_beep_delay_timer_.isTimerExpired()) {
         SaberBase::DoEffect(EFFECT_NEWFONT, 0);
     }
+    // Blade array changing helper. See id() for usage.
+    if (play_no_change_sound_) {
+      play_no_change_sound_ = false;
+
+      float real_id = PropBase::id(false);
+      size_t real_best_config = FindBestConfigForId(real_id);
+
+      if (real_best_config == 0) {
+        // Real blade event resulted in no blade array, so blade removal
```

Comment:
This comment doesn't make any sense.
real_best_config == 0 just means the first entry in the blade array.

Reply:
Maybe you wanted to check current_config->ohm >= NO_BLADE ?

Reply:
This is assumed that NO_BLADE is first in the BladeConfig.
If the blade is removed, that's where we land, so a blade must have been removed.
See comment below.

Reply:
> Maybe you wanted to check current_config->ohm >= NO_BLADE ?

This is good.
Done